src/components/SEO/index.js

```diff
@@ -0,0 +1,16 @@
+import { Helmet } from 'react-helmet-async';
+
+export const Static = function Static({ children }) {
+  return (
+    <Helmet>
+      <html lang="en" />
+      <title>Hello from React Helmet</title>
+      <meta name="description" content="Basic example" />
+      {children}
+    </Helmet>
+  );
+};
```

Comment:
Do we need this with such values?

Reply:
No. Sorry, we don't need Static component at the moment. I wasn't sure what I will use and forgot to delete it.

Reply:
```suggestion
```